emperor/support_files/js/color-view-controller.js

```diff
@@ -193,71 +204,154 @@ define([
    * continuous).
    */
   ColorViewController.getColorList = function(values, map, discrete, scaled) {
-    var colors = {}, numColors = values.length - 1, counter = 0;
-    var interpolator, min, max;
+    var scope = this;
```

Comment:
This is a class method, how is it that you need `scope` here? Is this vestigial from the refactoring? Also just noticed that `scaled` is being redefined below i.e. ti is part of the function's arguments and then is re-declared with var, since you are making changes to this file, would you mind changing that?